content/general/limits.textile

```diff
@@ -347,17 +391,38 @@ h3(#concurrent-functions). Function concurrency
 
 Function concurrency is the maximum number of functions that can run at the same time. This rate only applies to Google Cloud functions and Azure functions. AWS Lambda functions do not count towards this rate as they use the asynchronous @event@ invocation type.
 
+h3(#integration-warnings). Integration warnings
+
+Warnings for integrations are sorted into integration type and then warning type:
+
+* @reactor.httpEvent.maxRate.hard@ - when the hard limit on the account has been exceeded.
+* @reactor.httpEvent.maxRate.warning@ - when you reach 80% of your pre-paid quota.
+* @reactor.webhook.maxRate.hard@ - when the hard limit on the account has been exceeded.
+* @reactor.webhook.maxRate.warning@ - when you reach 80% of your pre-paid quota.
+* @reactor.externalQueue.maxRate.hard@ - when the hard limit on the account has been exceeded.
+* @reactor.externalQueue.maxRate.warning@ - when you reach 80% of your pre-paid quota.
+* @reactor.amqp.maxRate.hard@ - when the hard limit on the account has been exceeded.
+* @reactor.amqp.maxRate.warning@ - when you reach 80% of your pre-paid quota.
+
 h2(#api). API request limits
 
 API request limits are the maximum number of REST API requests that can be made to Ably. This excludes "token requests":#token.
 
 |_/2=. Limit |_\2=. Free |_\2=. Self-service |_\2=. Business (legacy) |_\2=. Enterprise |
 |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |
 | API requests (per hour) | 4,200 | 5,000 | 0.1 * hourly message soft limit | 0.1 * hourly message hard limit| 0.1 * hourly message soft limit | 0.1 * hourly message hard limit |\2. Custom |
-| API request rate (per second) | - | 20 | - | >=50 | - | >=50 |\2. Custom |
+| API request rate (per second) | - | 20 | - | 50 minimum | - | 50 minimum |\2. Custom |
 
 If the API request rate is exceeded, a "global instantaneous rate limit":#global-rates will be applied to API requests. The hourly API request limit is a "time-based limit":#time, meaning no additional API requests will be accepted until the following hour if the hard limit is hit.
 
+h3(#api-warnings). API warnings
+
+There are three types of apiRequest limit you may receive:
```

Comment:
I think we should be explicit that these relate to the hourly rate (even though it's in their names), rather than referring to them as `apiRequest` limits as this could be ambiguous still. 

There's also an `apiRequest.MaxRate.hard/warning` we should include for the per/second limit.